### PR TITLE
UI clarification

### PR DESCRIPTION
**Pull request by JoePastucha**
- Added information to the buddies page
- Added "Tap for more buddy info!" to entice users to tap their buddy to see their profile
- Added "Tap the day to log your progress!" to clear up issues from user tests where users were confused at how to log their progress

### CODE REVIEW

File: screens/buddies.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export default function Buddies({ navigation }) {
 
     return (
         <View style={dyStyles.buddyDisplayContainer}>
+            <Text style={dyStyles.title_text}>Tap your buddy for more info!</Text>
```
Comment (kny4) [inline screens/buddies.js:28]: How about "Select on a Buddy for more info!"? Maybe this is just me being nitpicky hahaha

**Review by kny4 — COMMENTED** [review]
I like it! Definitely makes the UI clearer for the clients

File: screens/buddies.js (hunk 1 above)
Comment (andrewJamesBaker) [reply screens/buddies.js:28]: I like what is there. I think having it explicitly say "Tap" is good. It shouldn't allow for any confusion.

**Review by andrewJamesBaker — APPROVED** [review]
Ran well for me. No issues. Will merge.

**Pull request merged**